src/main/proto/Payload.proto

```diff
@@ -10,6 +10,7 @@ message Payload {
     string queryString = 5;
     string body = 6;
     fixed64 flakeId = 7;
+    bool repeating = 8;
```

Comment:
What purpose does this serve?

Reply:
If you want the request to be repeating you need to set this bool. Its used in Katie

Reply:
Not a super-fan of calling this repeating but I have no better ideas.